### ISSUE TITLE
add similar function to spinMyR that works for Rmd docs (render/knitMyR)?

### ISSUE THREAD

**Issue by daattali** [daattali/rsalad#7]

**Comment by krlmlr** [comment]
Just went here to add this issue. Sometimes, I also want to use `.Rmd` as source file.

Looking at the code, it should be possible to extract the relevant parts that do the `.Rmd` -> `.md` -> `.html` conversion into a separate function (`knitMyRmd`?) and have `spinMyR` ultimately call `knitMyRmd`. Unfortunately, it doesn't look like there is an automated test for `spinMyR`, so any refactoring might wreak havoc...

**Comment by daattali** [comment]
Yes it shouldn't be too bad to add that functionality. I've taken a short hiatus from working on this because I was spending way too much time on my packages and not enough time on my graduate work, but I want to tie up the loose ends (such as this) soon :)

**Comment by daattali** [comment]
`spinMyR` is not staying in this package for long, and `knitMyR` already exists in its substitute [ezknitr](https://github.com/daattali/ezknitr)

**Comment by krlmlr** [comment]
Are you aware of `rmarkdown::render()`? This now seems to be the "official" way, and it looks like it supports R files as sources as well as properly processing files in subdirectories.

**Comment by daattali** [comment]
Thanks, I didn't know it was updated. I just by skimmed the arguments and it does look like they added some new ones recently that do take care of a few of the directory issues, so that's cool! I haven't looked into it too deeply, but it still looks like the working directory issues still exists, where the directory where the Rmd file lives is assumed to be the wd, and this was the main reason I started this little project... and there are a few more features I like here.  But it does look like they improved `render` a lot, maybe soon it will completely diminish my need to keep using this. Anyway thanks for the heads up Kirill
